demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -342,24 +335,12 @@ export const useCti = (
             },
             iframeLocation: data.iframeLocation,
           } as OnInitialized);
-          const incomingNumber =
-            window.localStorage.getItem(INCOMING_NUMBER_KEY);
-          const incomingContactName = window.localStorage.getItem(
-            INCOMING_CONTACT_NAME_KEY
-          );
-          if (engagementId && incomingNumber && incomingContactName) {
-            setEngagementId(engagementId);
-            cti.incomingNumber = incomingNumber;
-            setIncomingContactName(incomingContactName);
-            initializeCallingStateForExistingCall(incomingNumber);
-            // clear out localstorage
-            window.localStorage.removeItem(INCOMING_NUMBER_KEY);
-            window.localStorage.removeItem(INCOMING_CONTACT_NAME_KEY);
-          }
         },
         onDialNumber: (data: any, _rawEvent: any) => {
           const { phoneNumber } = data;
-          setPhoneNumber(phoneNumber);
+          if (phoneNumber) {
+            setDialNumber(phoneNumber);
+          }
```

Comment:
Whenever we receive the `DIAL_NUMBER` event from HubSpot, we set the phone number field.